test/support/file_endpoint.ex

```diff
@@ -0,0 +1,96 @@
+defmodule Membrane.RTC.Engine.Support.FileEndpoint do
+  @moduledoc false
+
+  # Endpoint that publishes data from a file in realtime. It will start publishing data when it receives message :start.
+  # Buffers from file must have timestamps, because it will be used in Realtimer.
+
+  use Membrane.Bin
+
+  alias Membrane.RTC.Engine
+
+  require Membrane.Logger
+
+  @type encoding_t() :: String.t()
+
+  def_options rtc_engine: [
+                spec: pid(),
+                description: "Pid of parent Engine"
+              ],
+              file_path: [
+                spec: Path.t(),
+                description: "Path to track file"
+              ],
+              track: [
+                spec: Engine.Track.t(),
+                description: "Track to publish"
+              ],
+              processing_children: [
+                spec: %{any() => Membrane.ParentSpec.child_spec_t()},
+                description: "Children required for processing input from file."
+              ],
+              processing_link: [
+                spec:
+                  (Membrane.ParentSpec.link_builder_t() -> Membrane.ParentSpec.link_builder_t()),
+                description: "Function which link source with processing children"
+              ]
```

Comment:
Maybe we could call this `interceptor`? 

https://en.wikipedia.org/wiki/Interceptor_pattern